h2o-py/h2o/estimators/aggregator.py

```diff
@@ -19,22 +19,59 @@ class H2OAggregatorEstimator(H2OEstimator):
     """
 
     algo = "aggregator"
-    param_names = {"model_id", "training_frame", "response_column", "ignored_columns", "ignore_const_cols",
-                   "target_num_exemplars", "rel_tol_num_exemplars", "transform", "categorical_encoding",
-                   "save_mapping_frame", "num_iteration_without_new_exemplar", "export_checkpoints_dir"}
 
-    def __init__(self, **kwargs):
+    def __init__(self,
+                 model_id=None,  # type: str
```

Comment:
Since `model_id` can be `None`, its type should be `Optional[str]`.  This also applies in all the other places where the parameter value can be `None`, not just optional attributes[1].

[1] https://docs.python.org/3/library/typing.html#typing.Optional